docs/PUBLISHING.md

```diff
@@ -93,7 +93,7 @@ access resin.io's production downloads S3 bucket.
 Run the following command to publish a specific file:
 
 ```sh
-./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -t <production|snapshot>
+./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -p <product name>
```

Comment:
I don't think I follow. What does documenting `-k` has to do with preventing snapshot builds from getting to master?

If you run `make publish-aws-s3`, it will always do the right thing based on the value of `RELEASE_TYPE`.

Someone could still publish somewhere else by using `./scripts/publish/aws-s3.sh`, so I guess the best thing to do would be to only describe `make publish-aws-s3`. Is that what you mean?